README.md

```diff
@@ -538,6 +538,40 @@ const Home = importComponent('./home', 'Home');
 export default render(<Home />);
 ```
 
+During tests `importComponent` simply outputs the referenced component. If you'd rather like to test the `loading`- or the `error`-state of the lazy-loading placeholder component, you currently have to provide your own mock.
+
+**Example: check the loading state**
+
+```javascript
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import testRenderer from 'react-test-renderer';
+import App from '../';
+
+jest.mock('hops', () => {
+  const hops = jest.requireActual('hops');
```

Comment:
Would it make sense providing the users with both `loading` and `error` state mocks to avoid possible future breaking changes related to how we mock/transpile this feature in tests?

Reply:
What do you mean?

Reply:
I have the impression that the suggested mocking strategy exposes a few **implementation details** of the currently **underlying `loadable` component** which might change in future: _a function returning  a `<Fragment>` which renders the result of `arguments.render`_.

Would a possible refactor of `loadable` component involve updating these custom mocks?

Reply:
I guess altering this behaviour would be a breaking change we will actually try to avoid. But providing ready-made mocks on the other hand unnecessarily extends the API surface we're responsible for. So it's actually easier for us to have the users mock the behaviour. While it's imho still a reasonable effort for the people out there.

But speaking of `Fragments`: they're actually not needed here. 😅